recipes/cuda-python/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "cuda-python" %}
+{% set version = "11.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e1c1bd49919c8646b7d84e3e63f5a0a971b64a69a9c986194d66f17cd26ee006
+
+build:
+  number: 0
+  skip: true  # [py<37 or osx or (cuda_compiler_version == "None")]
```

Comment:
```suggestion
  skip: true  # [py<37 or cuda_compiler_version in (undefined, "None")]
```

Reply:
Pushed a commit (I'm assuming removing osx from the selector was a typo).

Reply:
Thanks Mark! 😄

Oh it's ok to drop `osx` as `cuda_compiler_version` would be `undefined` or `"None"` there